src/services/webPlatformUtils.service.ts

```diff
@@ -1,5 +1,4 @@
-import * as _swal from 'sweetalert';
-import { SweetAlert } from 'sweetalert/typings/core';
+import Swal, {SweetAlertIcon} from 'sweetalert2/src/sweetalert2.js';
```

Comment:
This would not work due to the same reasons as below. It needs to stay precisely like in my commit. (This cost me a few minutes to figure out the first time)

Should I still change the linebreaks?
The result would look like this:
```
import Swal, {
    SweetAlertIcon,
} from 'sweetalert2/src/sweetalert2.js';
```

Reply:
Ok, just leave the line breaks as-is then. You need some spacing around the braces, though. 


```
import Swal, { SweetAlertIcon } from 'sweetalert2/src/sweetalert2.js';
```

Reply:
Done